### PR TITLE
MUHU-190: Added github actions

### PR DESCRIPTION
**Pull request by jekuaitk**
https://jira.itkdev.dk/browse/MUHU-190

* Added github actions

### CODE REVIEW

**Review by rimi-itk — APPROVED** [review]

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -79,3 +79,55 @@ jobs:
...
+
+      - name: 'Composer install with exported .env variables'
+        run: |
+          set -a && source .env && set +a
+          APP_ENV=prod composer install --no-dev -o
+      - name: Run Doctrine Migrations
+        run: APP_ENV=prod php bin/console doctrine:migrations:migrate --no-interaction
+
+      - name: Validate Doctrine schema
+        run: APP_ENV=prod php bin/console doctrine:schema:validate
```
Comment (rimi-itk) [inline .github/workflows/pr.yaml]: No newline at end of file.

File: src/Entity/YesplanEvent.php [hunk 2]
```diff
@@ -97,10 +97,10 @@ class YesplanEvent
     #[ORM\Column(nullable: true)]
     private ?bool $inPresaleDateUpdated = null;
 
-    #[ORM\Column]
+    #[ORM\Column(nullable: true)]
```
Comment (rimi-itk) [inline src/Entity/YesplanEvent.php:100]: Shouldn't this (`nullable: true`) be reflected in an updated migration?

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yaml (hunk 1 above)
Comment (jekuaitk) [reply .github/workflows/pr.yaml]: Added newline at end of file.

File: src/Entity/YesplanEvent.php (hunk 2 above)
Comment (jekuaitk) [reply src/Entity/YesplanEvent.php:100]: Nope, it was already this way in previous migrations, but when adding the entities `nullable: true` was forgotten.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**